python/lsst/ap/association/packageAlerts.py

```diff
@@ -276,7 +277,7 @@ def run(self,
                 objSourceHistory = diaSrcHistory.loc[srcIndex[0]]
             else:
                 objSourceHistory = None
-            if doRunForcedMeasurement and not diaForcedSources.empty:
+            if doRunForcedMeasurement and diaObject["nDiaSources"] > forcedSourceHistoryThreshold:
```

Comment:
The doc for historyThreshold is `Minimum number of detections of a diaObject required to run forced photometry. Set to 0 to include all diaObjects.` This requires  `diaObject["nDiaSources"] > forcedSourceHistoryThreshold`, which intuitively to me should be >=.